tests/test_fields.py

```diff
@@ -21,12 +23,28 @@
 )
 
 
+if PY3:
+    unicode = str
+
+
+class TestUtilsMixin(object):
+    def assertCountEqual(self, a, b):
+        """Assert a and b are equal in length."""
+        return len(a) == len(b)
+
+
+@contextmanager
+def sub_test(field):
+    """Implement sub_test."""
+    yield
```

Comment:
That was added by @protosac from what I can tell it works, but I'm happy to remove if it's easier.